### ISSUE TITLE
[Add new facility] Title during editing existing facility is not accurate

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6409]
## Area or Component

Settings > Manage Facilities > Existing Facility

## Quick Summary

When editing an existing facility, the page title is still "Add new facility"

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/title-not-meaningful

## Action Requested

If editing a facility, the page title should change

## Screenshots (optional)

![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/285b1bac-e4ef-4eaf-b5de-1e6f613908ff)

## Source Code (optional)

https://github.com/CDCgov/prime-simplereport/blob/d4404d89fd310a99382dbf7b2f50c366aae40a56/frontend/src/app/Settings/Facility/FacilityFormContainer.tsx#L23-L24
